### DISCUSSION TITLE
Windows Cursor Color

### DISCUSSION THREAD

**Discussion by pouchbunny** [enisn/UraniumUI#789] [General]
Can the cursor color be a bindable prop so we can set it to match the theme, or have it based on the Accent Color since it's the color being used when focusing on a textfield? When you put the text field on a frame the color of the cursor changes on its own and with my chosen color it's hard to see.

![image](https://github.com/user-attachments/assets/3dbe1ec2-86b2-4e26-a05d-bbf9e8373b01)

![image](https://github.com/user-attachments/assets/a4abae40-e987-4b94-a62f-603c671c6976)

**Comment by enisn** [comment]
I'll create an issue for that, probably Windows API has this customization, I need to expose it with a property
